.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+## Code Review Checklist
+ * Does the PR describe what changes are being made?
+ * Does the code follow the [Effective Go style guide](https://golang.org/doc/effective_go.html)?
+ * Does the new code have enough documentation? (comments and or diagrams)
```

Comment:
Also "Does any of the documentation need updating?"

Reply:
oh good call

Reply:
"enough documentation" is a little vague.  I'm personally curious, where do we expect people to be adding documentation? Seems like there are different places to do so.  For example there are two different wikis: https://wiki.storj.io/ and https://github.com/storj/storj/wiki (which apparently is sourced from the storj/docs repo).